src/App.tsx

```diff
@@ -1,25 +1,57 @@
-import React from 'react';
+import React, { useState } from 'react';
+
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
+
+const total = 42;
+
+function prepareItems(
+  itemsArray: number[],
+  itemPerPage: number,
+  currentPage: number,
+): number[] {
+  const firstNumber = (currentPage - 1) * itemPerPage + 1;
+  let lastNumber = itemPerPage * currentPage;
+
+  if (lastNumber > itemsArray[itemsArray.length - 1]) {
+    lastNumber = itemsArray[itemsArray.length - 1];
+  }
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42).map(n => `Item ${n}`);
+  const indexOfFirst = itemsArray.indexOf(firstNumber);
+  const indexOfLast = itemsArray.indexOf(lastNumber);
+
+  return itemsArray.slice(indexOfFirst, indexOfLast + 1);
+}
```

Comment:
Would be great to move this function to a separate file. As well as the constant